src/web/mjs/connectors/AzoraManga.mjs

```diff
@@ -1,6 +1,6 @@
-import WordPressMadara from './templates/WordPressMadara.mjs';
+import Connector from '../engine/Connector.mjs';
 
-export default class AzoraManga extends WordPressMadara {
+export default class AzoraManga extends Connector {
```

Comment:
Extending `AzoraWorld` may be another solution, which may not break bookmarks (in case of compatibility).

Reply:
Done